contrib/gce/sgc

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python
+# -*- coding: utf8 -*-
+
+
+from __future__ import absolute_import, division, print_function
+
+
+import sys
+import os
+import tempfile
+import argparse
+import subprocess
+
+
+def _search_file(filename, search_paths):
+    found_fn = None
+    for path in search_paths:
+        this_fn = os.path.join(path, filename)
+        if os.path.exists(this_fn):
+          found_fn = this_fn
+          break
+    return found_fn
+
+
+def _get_trunk(project, cmd):
+    trunk = _search_file("gcloud", os.environ["PATH"].split(":"))
+    if not trunk:
+        raise OSError("gcloud not found")
+    if project:
+        trunk += " --project=%s" % project
+    trunk += " " + cmd
+    return trunk
+
+
+def _call_command(tokens, verbose, dry_run):
+    cmd = " \\\n  ".join(tokens)
+    if verbose >= 0:
+        sys.stdout.write(cmd + "\n")
+    if not dry_run:
+        if verbose >= 0:
+            sys.stdout.write("sending command ... \n")
+        tokens = " ".join(tokens).split()
+        subprocess.check_call(tokens)
+
+
+class _SubCommandFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                           argparse.RawDescriptionHelpFormatter):
+    pass
+
+
+class _CreateInstance(object):
```

Comment:
This command creates an instance using either (i) upstream Ubuntu 14.04 LTS image or (ii) a custom image created by this command.